laser_encoders/models.py

```diff
@@ -83,6 +91,11 @@ def __init__(
         self.encoder.eval()
         self.sort_kind = sort_kind
 
+    def __call__(self, sentences):
+        if self.spm_model:
```

Comment:
If there is no `spm_model`, this method will silently return `None`. 
I suggest raising instead an error that explains that either the encoder should be initialized with an `spm_model` for this method to work, or the sentences must be pre-tokenized and passed to `encode_sentences`.